docs/README.md

```diff
@@ -0,0 +1,22 @@
+# Ivadomed Docs
```

Comment:
out-of-scope? or maybe it includes a recently-merged branch?

Reply:
Ah yes, I added the README...should I make a separate PR for that?

Reply:
isn't the installation covered in https://github.com/ivadomed/ivadomed/pull/561/files?

Reply:
The installation doesn't go over how to build the `Sphinx` docs, and the `requirements.txt` is missing one of the `Sphinx` dependencies.

Reply:
I have removed the `README` from this pull request though